FABulous/fabric_definition/Gen_IO.py

```diff
@@ -0,0 +1,57 @@
+from dataclasses import dataclass, field
+from FABulous.fabric_definition.define import IO
+
+
+@dataclass
+class Gen_IO:
```

Comment:
I think you can make this can inherit the Bel class, which will allow this to treat it as normal bel as well.

Reply:
This will still be a single Bel. It will just have a special attribute.  

So rather than:
```
module IO_bidir_1(...);
endmodule
```
and you have lot of them,
become
```
module IO_bidir_3(...);
IO_bidir_1 inst_1(...);
IO_bidir_1 inst_2(...);
IO_bidir_1 inst_3(...);
```

but still a single bel.

Reply:
If this is how it was designed in the first place, then leave it as it is. Since it will eventually become a Bel, this does not matter then.